src/srt_congest_ctrl.rs

```diff
@@ -13,3 +13,10 @@ impl CongestCtrl for SrtCongestCtrl {
         10_0000
     }
 }
+
+impl SrtCongestCtrl {
+    #[allow(non_snake_case)]
+    pub fn SDN(&self) -> Duration {
```

Comment:
I understand, send_interval and SDN were the same thing, it was an experiment to provoke discussion. How close can I make the code look like the spec. Excess or non-essential complexity is the distance between the concepts/model of the domain and the concepts and code in the implementation. Using as close to the same language in the code as is in the model can help us differentiate one from the other. In this case the spec and ref-implementation guide what the domain model is. The fact that it wasn't part of the trait was due to expediency. I would also suggest we avoid both ad-hoc and parametric polymorphism in the core domain model. Although I haven't gone too deep into the congestion control algorithms yet, combination of cheap sum types (enums) and pattern matching can actually provide some benefits over polymorphism. I replaced the ad-hoc polymorphism you were using in the mid session connection handshaking implementation with an enum. IMO it's more clear, explicit, because there are less layers of non-essential indirection.